samples/sockets/client.cpp

```diff
@@ -594,7 +594,7 @@ void DoDownload(const wxString& urlname)
 
     // Try to get the input stream (connects to the given URL)
     wxLogMessage("Establishing connection to \"%s\"...", urlname);
-    const std::auto_ptr<wxInputStream> data(url.GetInputStream());
+    const wxScopedPtr<wxInputStream> data(url.GetInputStream());
```

Comment:
We should include `wx/scopedptr.h` from here now (but, to compensate, we probably can remove the `#include <memory>`).